docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,306 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
+########################################
+
+With the Courses API **Courses** resource, you can complete the
+following tasks.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+.. _Get the list of courses visible to a user:
+
+*****************************************
+Get a List of Courses Visible To a User
+*****************************************
+
+The endpoint to get a list of courses that are visible to a specified user is
+``/api/courses/v1/courses/``.
+
+=====================
+Use Case
+=====================
+
+Get a list of courses that are visible to a specified user.
+If the request is made by an anonymous user, a username is not required.
+
+=====================
+Example Request
+=====================
+
+GET /api/courses/v1/courses/?&username=<username>
+
+GET /api/courses/v1/courses/
+
+=====================
+Parameters
+=====================
+
+* username (optional) - The username of the user for whom the course data is
+  being accessed. If the request is made by an anonymous user, username is not
+  required.
+
+* org (optional) - A code for an organization; case-insensitive. Example:
+  "HarvardX". If ``org`` is specified, the list of courses is filtered such
+  that only those courses belonging to the specified organization are
+  returned.
+
+* mobile (optional) - If specified, the list of courses is filtered such that
+  only those courses that are designated as ``mobile_available`` are returned.
+
+=====================
+Response Values
+=====================
+
+The response for ``GET /api/courses/v1/courses/`` consists of the
+following fields as returned by `CourseListView`.
```

Comment:
No need to mention the Python class. The other sections introduce the list of values with:

The following fields are returned with a successful response.